federatedscope/vertical_fl/xgb_base/worker/train_wrapper.py

```diff
@@ -62,7 +62,7 @@ def _find_and_send_split(self, split_ref, tree_num, node_num):
                 self.model[tree_num][node_num].member = client_id
 
                 split_ref['feature_idx'] -= accum_dim
-                split_child = False
+                split_child = 0
```

Comment:
I suggest preserving `True/False` here to make it readable, and only transforming the type when `split_child` is in `message.content`